packages/reporting-server/rest_server/app.py

```diff
@@ -51,7 +62,7 @@ def get_app(run_config=SystemMode.ALL):
         app,
         db_url=app_config.db_url,
         modules={"models": ["models"]},
-        generate_schemas=run_config in (SystemMode.ALL, SystemMode.FLUENTD),
+        generate_schemas=False,
```

Comment:
I have some confusion when I was working with tortoise-orm. The docs says to not run `generate_schemas` as "part of the application". But in a fresh start without any existing database, we need to run `generate_schemas` to create the tables.

What is your approach to handling table creation? Is it always done by aerich in the pre-startup script?

Reply:
Yeah, we are now dropping the `generate_schemas` which is a fast built-in migration tool from Tortoise and we are moving completely to aerich to have more control over the migration and versions. So we need to create a database manually and then run the migrations `aerich upgrade` (because we already have the migration sql files)